src/apiCalls.js

```diff
@@ -26,25 +28,29 @@ const getData = () => Promise.all([fetchGuestData(), fetchRoomData(), fetchBooki
 
 
 
-export const postData = () => {
+const postData = (user, dateSelected, roomNum) => {
   return fetch(`http://localhost:3001/api/v1/bookings`, {
     method: 'POST',
-    body: JSON.stringify(booking),
+    body: JSON.stringify({
+      userID: user.id,
+      date: dateSelected,
+      roomNumber: roomNum,
+    }),
     headers: {
       'Content-type': 'application/json'
     }
   })
-  .catch(err => console.log(err))
-  // .then(checkForError)
+  .then(checkForError)
+  .then(() => loadPageInfo())
+  .catch(err => console.error(err))
 }
 
 const checkForError = (response) => {
   if(!response.ok) {
-    // dom update to show error???
     throw new Error('Something went wrong')
```

Comment:
nice check for error! You will want to add that dom stuff at some point to show the user if their booking didn't go through